website/templates/emails/file_updated.html.mako

```diff
@@ -6,6 +6,23 @@
             <span class="content" style="display: block;padding: 6px 5px 0px 8px;font-size: 14px;">
                 <span class="person" style="font-weight: bold;">${user.fullname} </span>
                 ${message}
+                % if storage_limit_context and admin_recipient:
+                    % if storage_limit_context['public']:
+                        This public project is ${storage_limit_context['storage_limit_status']} the 50GB OSF Storage limit and requires your attention. In order to avoid disruption of your workflow, please take action through one of the following options:<br>
+                        <ul>
+                            <li>Connect an <a href="https://help.osf.io/hc/en-us/sections/360003623833-Storage-add-ons">OSF Storage add-on</a> to continue managing your research efficiently from OSF. OSF add-ons are an easy way to extend your storage space while also streamlining your data management workflow.</li>
+                            <li><a href="https://help.osf.io/hc/en-us/articles/360019737614-Create-Components">Organize your project with components</a> to take advantage of the flexible structure and maximize storage options.</li>
+                        </ul>
+                    % else:
+                        This private project is ${storage_limit_context['storage_limit_status']} the 5GB OSF Storage limit and requires your attention. In order to avoid disruption of your workflow, please take action through one of the following options:<br>
+                        <ul>
+                            <li>Connect an <a href="https://help.osf.io/hc/en-us/sections/360003623833-Storage-add-ons">OSF Storage add-on</a> to continue managing your research efficiently from OSF. OSF add-ons are an easy way to extend your storage space while also streamlining your data management workflow.</li>
+                            <li>Free up space by making one or more of your components public. <a href="https://help.osf.io/hc/en-us/articles/360019737614-Create-Components">Organize your project with components</a> to take advantage of the flexible structure and maximize storage options.</li>
```

Comment:
Should the "organize your project" bit be its own list entry?

Reply:
It should be combined! Here's the doc I adapted the language from (https://docs.google.com/document/d/1gB5Ya4InmZ3ozOQND1E-drVxnwYJZmzlfV1ohqwr-kQ/edit#); there are some modifications because we decided to include the storage messaging in file notifications.